azure/services/managedclusters/managedclusters.go

```diff
@@ -43,8 +47,12 @@ type ManagedClusterScope interface {
 	SetControlPlaneEndpoint(clusterv1.APIEndpoint)
 	SetKubeletIdentity(string)
 	MakeEmptyKubeConfigSecret() corev1.Secret
-	GetKubeConfigData() []byte
-	SetKubeConfigData([]byte)
+	GetAdminKubeConfigData() []byte
```

Comment:
```suggestion
	GetAdminKubeconfigData() []byte
```
For consistency of the capitalization through the repo

Reply:
ack